scene/main/window.cpp

```diff
@@ -2635,7 +2635,7 @@ void Window::set_unparent_when_invisible(bool p_unparent) {
 
 void Window::set_layout_direction(Window::LayoutDirection p_direction) {
 	ERR_MAIN_THREAD_GUARD;
-	ERR_FAIL_INDEX((int)p_direction, 4);
+	ERR_FAIL_INDEX((int)p_direction, 5);
```

Comment:
We probably should have `LAYOUT_DIRECTION_MAX` to avoid setting it manually.

Reply:
`LAYOUT_DIRECTION_MAX` added.